### ISSUE TITLE
able to use Scent while eating/drinking

### ISSUE THREAD

**Issue by RagdollZombie** [TheIsleGame/bugs#220]
## Expected Behavior

Scent should either cancel the eating/drinking event, or you shouldn't be able to use scent while eating/drinking

## Current Behavior

If you activate scent while eating/drinking, the sniffing animation will play but you will still gain hunger/thirst from eating/drinking and scent will still highlight food and footprints.

## Steps to reproduce

Spawn as any dinosaur
Find a food or water source
Start eating or drinking, then hold down the scent button
Scent animation will override eating/drinking animation, but you will still gain hunger and thirst. Scent will also still highlight objects when the sniffing animation finishes. 

Basically players could use this to eat and smell at the same time.

## Build Information

> **Game Version:** 0.1.38.8

> **Operating System**: Windows

> **Network Role**: Client

**Comment by looterz** [comment]
Fixed in 0.1.39.0
